compiler/one-cmds/one-infer

```diff
@@ -205,6 +208,15 @@ def main():
 
     # run backend driver
     _utils._run(infer_cmd, err_prefix=ntpath.basename(driver_path))
+    
+    # run post process script if it's given
+    if _utils._is_valid_attr(args, 'post_process'):
+        # NOTE: the given python script will be executed by venv of ONE
+        python_path = sys.executable
+        post_process_command = [python_path] + getattr(args, 'post_process').strip().split(' ')
+        print(f'one-infer: [post-process] {post_process_command[1]} is started', flush=True)
```

Comment:
I think `print` can be omitted if not really necessary.

Reply:
`is started` is somewhat strange. maybe `has started` or `will start`, ...
or just `begin`, `start`

Reply:
I'll remove the `print`s.